code/modules/power/antimatter/control.dm

```diff
@@ -239,6 +240,9 @@
 
 /obj/machinery/power/am_control_unit/proc/toggle_power(powerfail = 0)
 	active = !active
+	investigate_log("turned [active ? "ON" : "OFF"] by [usr ? key_name(usr) : "outside forces"] at [AREACOORD(src)]", INVESTIGATE_ENGINES)
+	message_admins("Antimatter turned [active ? "ON" : "OFF"] by [usr ? ADMIN_LOOKUPFLW(usr) : "outside forces"] in [ADMIN_VERBOSEJMP(src)]")
+	log_game("Antimatter turned [active ? "ON" : "OFF"] by [usr ? "[key_name(usr)]" : "outside forces"] at [AREACOORD(src)]")
```

Comment:
Double logging bad, just log it in engines + message

Reply:
This is how the particle accelerator does it

Reply:
Log game seems unnecessary in both cases